src/App.tsx

```diff
@@ -1,54 +1,123 @@
-import React from 'react';
+import React, { useCallback, useState } from 'react';
 import './App.scss';
+import classNames from 'classnames';
+import debounce from 'lodash.debounce';
```

Comment:
Please try to fix this import error
![image](https://github.com/mate-academy/react_autocomplete/assets/54936044/b882f8e8-7a6c-48b5-85eb-8fbb294375e0)